### ISSUE TITLE
Search does not close in 5.0.0 RC 1

### ISSUE THREAD

**Issue by cbenhagen** [squidfunk/mkdocs-material#1505]
__I checked that...__

- [X] ... the documentation does not mention anything about my problem
- [X] ... the problem doesn't occur with the default MkDocs template
- [X] ... the problem is not in any of my customizations (CSS, JS, template)
- [X] ... there are no open or closed issues that are related to my problem

## Description

When clicking on a search result, the search results list is not being closed.

### Package versions

* Python: 3.8.1
* MkDocs: 1.1
* Material: 5.0.0rc1

### Project configuration

``` yaml
site_name: "test"
theme:
  name: 'material'
  language: 'en'

markdown_extensions:
  - admonition
  - attr_list
  - pymdownx.betterem:
      smart_enable: all
  - pymdownx.caret
  - pymdownx.critic
  - pymdownx.emoji:
      emoji_generator: !!python/name:pymdownx.emoji.to_svg
  - pymdownx.inlinehilite
  - pymdownx.magiclink:
      hide_protocol: True
  - pymdownx.mark
  - pymdownx.smartsymbols
  - pymdownx.superfences
  - pymdownx.tasklist:
      custom_checkbox: true
  - pymdownx.tilde
```

### System information

* OS: macOS 10.15.3
* Browser: Safari / Chrome

**Comment by squidfunk** [comment]
Confirmed, thanks!

**Comment by squidfunk** [comment]
Fixed in c0abaf56. The JavaScript code still needs some refactoring, but it should work for now.

**Comment by squidfunk** [comment]
Just released RC 2. Please re-open if the issue persists.
